cog/version_info.py

```
@@ -82,53 +51,70 @@ def embed_installed_software(self) -> discord.Embed:
+        # version
+        #
+        # '''pip''' <!-- To-do -->
+        # version
+        #
+        # '''mysql''' <!-- To-do -->
+        # version
+        #
+        # == Installed packages == <!-- To-do -->
+        description = (
+            "### 已安裝的軟體\n\n"
+            + "**中電喵**\n"
```

Ruff (0.14.7)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)